Practice2/README.md

```diff
@@ -34,9 +34,18 @@
     * images/
   * styles/
```

Comment:
How about the fonts? I don't see any definition of font face in the skeleton.

Reply:
@duylong13 This is still not resolved. Search with keyword `font face` to refer.

Reply:
Added

Reply:
@duylong13 Still don't see any font face defined.

Reply:
Defined

Reply:
@duylong13 Thank you. The font face is for typography, so should it be placed in the `variables` folder?

Reply:
Since I already set it to base.css, I don't think this is necessary